codegen/Cargo.toml

```diff
@@ -1,10 +1,7 @@
 [package]
 name = "codegen"
-authors = ["Lucio Franco <luciofranco14@gmail.com>"]
-license = "MIT"
```

Comment:
I'm actually not sure about removing these licenses. If people want to copy/paste the example code, they should feel free too, and I don't think that would be "allowed" if we don't specify a license under which it is allowed.

Reply:
Fair enough, but I think you would need copyright notice and spdx license headers for that. To me it seems this field is intended more for importing crates rather than specifying how files are licensed under a subfolder. There is an overall license on the overall git repo. The usage of `license` is also a bit inconsistent, where `tests/service_named_result/Cargo.toml` did not have any mentioned. So which crates you would want to maintain the `package.license` field?

Reply:
Basically the `examples` one.